bin/sgqlc-codegen

```diff
@@ -324,12 +324,15 @@ default=%(default)s)),
 
     def write_type_input_object(self, t):
         name = t['name']
+        field_names = [self.graphql_to_python(f['name']) for f in t['inputFields']]
```

Comment:
make this a tuple: `tuple(... for f in t... )`

Reply:
Fixed in a following commit.